server/routes/discover.ts

```diff
@@ -106,10 +153,18 @@ discoverRoutes.get<{ genreId: string }>(
 
 discoverRoutes.get<{ studioId: string }>(
   '/movies/studio/:studioId',
-  async (req, res) => {
+  async (req, res, next) => {
     const tmdb = createTmdbWithRegionLanaguage(req.user);
 
-    const studio = await tmdb.getStudio(Number(req.params.studioId));
+    const studio = await tmdb
+      .getStudio(Number(req.params.studioId))
+      .catch(async () => {
+        return undefined;
+      });
```

Comment:
Not sure I like this. Might be better to wrap this in a try/catch?

Reply:
So that's what I tried initially, but it wasn't working :|  I'll play around with it some more and see if I can figure out another way, though!

Reply:
@sct Okay, not sure what was going on before but I switched it back to the try-catch block and it just worked this time?  I must have done something wrong earlier haha